src/control-manager.cpp

```diff
@@ -276,14 +294,12 @@ namespace dynamicgraph
             /// if not use the default value
             if(m_max_currentSIN.isPlugged())
               m_maxCurrent = m_max_currentSIN(iter)(i);
-            else
-              m_maxCurrent = DEFAULT_MAX_CURRENT;
 
             if( (fabs(pwmDes(i)) > m_maxCurrent) || 
                 (fabs(s(i))      > m_maxCurrent * FROM_CURRENT_TO_12_BIT_CTRL) )
             {
               m_emergency_stop_triggered = true;
-              SEND_MSG("Joint "+JointUtil::get_name_from_id(i)+" desired current is too large: "+
+              SEND_MSG("Joint "+m_robot->model().getJointName(i)+" desired current is too large: "+
```

Comment:
Fix in commit bc5f539678009aa87d3eaa2a26e524133f5d03dd. But IMHO adding a remapping between name and id in a complex robotics application, in the long run, may just generate another set of confusion for newcomers...